#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_fix_smarty_location.dpatch by  <cajus@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Adapt location for debian packaged smarty

@DPATCH@
diff -urNad gosa-core~/include/php_setup.inc gosa-core/include/php_setup.inc
--- gosa-core~/include/php_setup.inc	2008-03-17 23:52:36.000000000 +0100
+++ gosa-core/include/php_setup.inc	2008-04-07 23:12:32.000000000 +0200
@@ -274,7 +274,7 @@
 ini_set("include_path",".:$BASE_DIR/include:$BASE_DIR/include/utils/excel");
 
 /* Do smarty setup */
-require("smarty/Smarty.class.php");
+require("/usr/share/php/smarty/libs/Smarty.class.php");
 $smarty = new Smarty;
 $smarty->template_dir = $BASE_DIR.'/ihtml/';
 $smarty->caching= false;
